### PR TITLE
fix to graph simplifier 

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/aaVUAALc/357-graph-simplified-line-looks-strange

### CODE REVIEW

File: AirCasting/Graph/Graph.swift [hunk 1]
```diff
@@ -73,12 +73,16 @@ struct Graph: UIViewRepresentable {
         
         let counter: Int = entries.filter({ $0.x >= startTime && $0.x <= endTime }).count
         
-        let simplifiedPoints = AirCastingGraphSimplifier.simplify(points: entries,
-                                                             visibleElementsNumber: counter,
-                                                             thresholdLimit: simplifiedGraphEntryThreshold)
-        
-        uiView.updateWithEntries(entries: simplifiedPoints, isAutozoomEnabled: isAutozoomEnabled)
-        print(entries.count != simplifiedPoints.count ? "Simplified \(entries.count) to \(simplifiedPoints.count)" : "Not simplyfing")
+        if counter > simplifiedGraphEntryThreshold {
```
Comment (pawelgil) [inline AirCasting/Graph/Graph.swift:76]: if/else blocks at the end for a function can be nicely simplified using `guard`

```Swift
guard counter > simplifiedGraphEntryThreshold else {
    uiView.updateWithEntries(entries: entries, isAutozoomEnabled: isAutozoomEnabled)
    return
}
let simplifiedPoints = AirCastingGraphSimplifier.simplify(points: entries, visibleElementsNumber: counter, thresholdLimit: simplifiedGraphEntryThreshold)
uiView.updateWithEntries(entries: simplifiedPoints, isAutozoomEnabled: isAutozoomEnabled)
print("Simplified \(entries.count) to \(simplifiedPoints.count)")
```

File: AirCasting/Graph/AirCastingGraphSimplifier.swift [hunk 2]
```diff
@@ -2,16 +2,15 @@
 //
 
 import Foundation
-import SwiftSimplify
 import Charts
 
 final class AirCastingGraphSimplifier {
     
-    static func simplify<P: Point2DRepresentable>(points: [P], visibleElementsNumber: Int, thresholdLimit: Int) -> [P] {
+    static func simplify(points: [ChartDataEntry], visibleElementsNumber: Int, thresholdLimit: Int) -> [ChartDataEntry] {
```
Comment (pawelgil) [inline AirCasting/Graph/AirCastingGraphSimplifier.swift:9]: is this de-generalization really needed? It doesn't look like it is changing anything except, well, de-generalizing ;)

Comment (jankrzempek) [reply AirCasting/Graph/AirCastingGraphSimplifier.swift:9]: Yeah, you right. There is one thing - the  `Point2DRepresentable` was taken from SwiftSimplifier library which we have added. I thought that if we are going to use our own implementation now, we can get rid of the library in the future. That was the point of changing it, what do you think?

**Review by anna1901 — APPROVED** [review]
Line looks good now 🚀

File: AirCasting/Graph/AirCastingGraphSimplifier.swift (hunk 2 above)
Comment (pawelgil) [reply AirCasting/Graph/AirCastingGraphSimplifier.swift:9]: We could create our own protocol for that, but in this case I think it wouldn't matter. Cool then! 😀

**Review by pawelgil — APPROVED** [review]

**Pull request merged**